FSDigitaltechnik.tex

```diff
@@ -172,28 +170,32 @@ \section{Boolsche Algebra}
 	Konfiguration: $f = c_1 + c_2 + c_3 \Ra cov(f) = \eset{c_1, c_2, c_3}$
 
 	\subsection{Gesetze der booleschen Algebra}
-	\begin{tabular}{l|l|l}
-		            & Boolesche Algebra                                     & Mengenalgebra                                         \\
-		            & $({0,1};\cdot , +, \overline{x})$                     & $(P(G);\cap , \cup, \overline{A};G,\emptyset )$       \\ \hline
-		Kommutativ  & $x \cdot y = y \cdot x$                               & $A \cap B = B \cap A$                                 \\
-		            & $x + y = y + x$                                       & $A \cup B = B \cup A$                                 \\
-		Assoziativ  & $x \cdot (y \cdot z) = (x \cdot y) \cdot z$           & $(A \cap B) \cap C = A \cap (B \cap C)$               \\
-		            & $x + (y + z) = (x + y) + z$                           & $(A \cup B) \cup C = A \cap (B \cup C)$               \\
-		Distributiv & $x \cdot (y + z) = x \cdot y + x \cdot z$             & $A \cap (B \cup C) = (A \cap B) \cup (A \cap C)$      \\
-		            & $x + (y \cdot z) = (x + y) \cdot (x + z)$             & $A \cup (B \cap C) = (A \cup B) \cap (A \cup C)$      \\
-		Indempotenz & $x \cdot x = x$                                       & $A \cap A = A$                                        \\
-		            & $x + x = x$                                           & $A \cup A = A$                                        \\
-		Absorbtion  & $x \cdot (x+y) = x$                                   & $A \cap (A \cup B) = A$                               \\
-		            & $x + (x \cdot y) = x$                                 & $A \cup (A \cap B) = A$                               \\
-		Neutral     & $x \cdot 1 = x$                                       & $A \cap G = A$                                        \\
-		            & $x + 0 = x$                                           & $A \cup \emptyset = A$                                \\
-		Dominant    & $x \cdot 0 = 0$                                       & $A \cap \emptyset = \emptyset$                        \\
-		            & $x + 1 = 1$                                           & $A \cup G = G$                                        \\
-		Komplement  & $x \cdot \overline{x} = 0$                            & $A \cap \overline{A} = \emptyset$                     \\
-		            & $x + \overline{x} = 1$                                & $A \cup \overline{A} = G$                             \\
-		            & $\overline{\overline{x}} = x$                         & $\overline{\overline{A}} = A$                         \\
-		De Morgan   & $\overline{x \cdot y} = \overline{x} + \overline{y}$  & $\overline{A \cap B} = \overline{A} \cup \overline{B}$\\
-		            & $\overline{x + y} = \overline{x} \cdot \overline{y}$ & $\overline{A \cup B} = \overline{A} \cap \overline{B}$
+	\begin{tabular}{l|l}
+		Gesetz & Boolesche Algebra                                        \\
```

Comment:
Warum die Mengenalgebra raus?

Reply:
Haben die Rechenregeln in VL und ZÜ nie mit Mengenalgebra gelernt, immer nur + und Punkt. Lediglich die Definitionen der Standardgatter enthielten auch Notationen mit logisch "UND" und "ODER", das war aber auch alles was wir dazu gemacht haben soweit ich weiß. Ich kanns aber auch wieder reinmachen.

Reply:
Machs gerne wieder rein. Im Idealfall wird die Formelsammung allgemeiner und overfitet nicht zu sehr auf ein Semester

Reply:
Alles klar